akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ClusterShardingHealthCheck.scala

```diff
@@ -4,22 +4,18 @@
 
 package akka.cluster.sharding
 
-import akka.actor.ActorRef
-import akka.actor.ActorSystem
-import akka.annotation.ApiMayChange
+import scala.concurrent.duration.FiniteDuration
+import scala.concurrent.{ExecutionContext, Future}
```

Comment:
scalafmt is failing this PR, I suspect because our scalafmt expects spaces inside the curly's.